ui-e2e/envs/erc-contracts.env

```diff
@@ -33,3 +33,5 @@ ERC20_TOKEN_ADDRESS=0x3C665A31199694Bf723fD08844AD290207B5797f
 
 REQUIRED_NUMBER_OF_VALIDATORS=1
 VALIDATORS="0xaaB52d66283F7A1D5978bcFcB55721ACB467384b"
+
+DEPLOYMENT_GAS_LIMIT=5000000
```

Comment:
you already have this parameter in the line 4 but with a different value.

Reply:
Done.